src/config.js

```diff
@@ -1,3 +1,11 @@
+// Here are some badges we set
+import goldBrain from './assets/badges/goldBrain.svg';
```

Comment:
I think eventually we want to move away from the config as a `.js` file and move towards a `.json` or `.yaml` file. How about instead of imports we do URLs within the object that's being exported?

Reply:
I agree, that was more of a temporary change for our implementation.